### DISCUSSION TITLE
Initialization problem

### DISCUSSION THREAD

**Discussion by wcizmowski** [dunglas/symfony-docker#343] [General]
Cześć,
On my Linux is problem.

#Error:
``
16:16 $ sudo docker-compose build --pull --no-cache
``
Building php
Sending build context to Docker daemon  18.94kB
Error response from daemon: dockerfile parse error line 23: Unknown flag: link
ERROR: Service 'php' failed to build : Build failed

I admit I found a similar post saying that the version of docker engine itself is too low.
Should I update docker? docker-compose ?

**Comment by maxhelias** [comment]
Yes, your version is too low and use `docker compose` instead of `docker-compose` for use the version 2

**Reply by wcizmowski** [reply]
I have Linux, after command docker compose I have:

17:12 $ sudo docker compose build
docker: 'compose' is not a docker command.
See 'docker --help'

P.S.
On Windows 10 I don't have problem with this project, but I need this environment  on Linux
Please help

**Reply by QuentinDamianino** [reply]
That's because you have installed docker compose 1.x which has command `docker-compose.`

You have to install docker compose v2.x to use command `docker compose`

**Comment by wcizmowski** [comment]
hi,
The problem was solved, I updated the docker version.
